__tests__/routes/adminIncidents.test.js

```diff
@@ -1,17 +1,31 @@
 const request = require('supertest');
-const db = require('../../data/db-config');
-const Incidents = require('../../api/allIncidents/incidentsModel');
-const server = require('../../api/allIncidents/adminIncidentsRouter');
+const server = require('../../api/app');
+const authRequired = require('../../api/middleware/authRequired');
+jest.mock('../../api/middleware/authRequired');
+
+// Allows you to pass Admin authentication
+beforeEach(() => {
```

Comment:
Nice work figuring out how to mock the authorization to run the tests! This is a nice solution!